Added README.md and CONTRIBUTING.md (#9)
eclipse/che-workspace-loader: CONTRIBUTING.md

```bash
# [ws-loader-dev container]
$ cd /projects/che-workspace-loader && yarn build
```

### Run tests

In order to run unit tests you need to find and run task `[WS] run tests` in `My workspace` panel or execute `yarn test` command in project directory:

```bash
# [ws-loader-dev container]
$ yarn test
```

### Start development server

The easiest way to start dev server is to run task `[WS] start dev server` from `My workspace` panel. Under the hood it executes following command in project directory:

```bash
# [ws-loader-dev container]
$ yarn start --disable-host-check --public=$(echo ${server.dev-server} | sed -e s/https:\\/\\/// -e s/http:\\/\\/// -e s/\\///) --host="0.0.0.0" --env.target=${CHE_API_EXTERNAL%????}
```

### Testing new workspace loader

When workspace loader server is run, you probably need to test introduced changes.

To safely test the changes it is better to create a separate workspace which will be used by new loader.
But by default, Che doesn't allow to run more than one workspace simultaneously.
To change this behaviour you need to set `che.limits.user.workspaces.run.count` Che property to value greater than `1`.

> In development environment that could be reached by adding `CHE_LIMITS_USER_WORKSPACES_RUN_COUNT` environment variable for Che server deployment.

Reviewer: ```suggestion
In development environment that could be reached by adding `CHE_LIMITS_USER_WORKSPACES_RUN_COUNT` environment variable for Che server config map.
```
But since operator will rewrite these changes it makes sense to add just a reference to documentation where it's described how to change property depending on deployment method.

Also for developers, it makes sense to paste here simple commands:
```
kubectl patch configmap che --patch "{\"spec\":{\"data\":{\"CHE_LIMITS_USER_WORKSPACES_RUN_COUNT\": \"-1\"}}}" --type=merge -n che
kubectl rollout restart deployment/che -n che
```

Please note, after changing deployment config you need to apply changes by rolling out (or rescaling) the corresponding pod

> (in case of Openshift just add the environment variable via Openshift dashboard in the `Environment` tab of the Che server deployment and the pod will be rolled out automatically).

Reviewer: ```suggestion
(in case of OpenShift just add the environment variable via Openshift dashboard in the `Environment` tab of the Che server deployment and the pod will be rolled out automatically).
```

To be able to point new workspace loader to the test workspace it is required to add the the test workspace id to the path of workspace loader route.
So, first, we need to retrieve the test workspace id.

> This could be done using swagger (please note, it might be disabled on production environment).

Reviewer: ```suggestion
This could be done using swagger UI.
```

we finally has swagger UI on che.openshift.io as well.

To open swagger just open Che dashboard and replace the path with `swagger`.
Then navigate to `workspace` section `GET /workspace` method.
It will return all user workspaces.
Find the test workspace id.
Second, to modify the path of the workspace loader server uri, retrieve the route of the server.
To do it, find workspace loader dev workspace id from the query in swagger above and use it as a key in `GET /workspace/{key}` method.
From the response get the workspace loader server url

> (if using the given defile for workspace loader development it should be under `runtime.machines.ws-loader-dev.serevrs.dev-server.url` key).

Reviewer: ```suggestion
(if using the given defile for workspace loader development it should be under `runtime.machines.ws-loader-dev.servers.dev-server.url` key).
```

The URI of workspace loader pointed to the test workspace should look like: `<workspace-loader-route>/<test-workspace-id>`.
For example: `http://server60zomi2d-dev-server-3000.192.168.99.100.nip.io/workspaceztcx9u432labmvxi` or `http://routeu5efcg53-che.apps-crc.testing/workspaceztcx9u432labmvxi` (depending on the infrastructure on which Che is run).

> In most cases multiuser Che is deployed.

Reviewer: ```suggestion
In most cases, Multi-User Che is deployed.
```

> To permit all the required connections it's needed to edit Keycloak settings.

Reviewer 2: I think we should provide the simplest method to open the Keycloak admin page. If I am not mistaken, the easiest way from the UD. Open an Account, then click the 'Edit' button.  If you have the needed rights, it will be green and Keykloak admin page will be opened after the click.

Am I right?

Author: it will open an account page, not the admin one.
